src/utils/loadTransportPlugin.ts

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: Apache-2.0
+import { loggerService } from '..';
+import { ITransportClass } from '../interfaces/ITransportPlugin';
+
+export const loadTransportPlugin = async (pluginName: string): Promise<ITransportClass> => {
+  try {
+    // Dynamically import the plugin
+    loggerService.log(`Loading plugin ${pluginName}`);
+    const { default: plugin } = await import(`${pluginName}`);
```

Comment:
Why is string interpolation needed here? Can't I just:
`await import(pluginName)`?